pages/_app.tsx

```diff
@@ -4,10 +4,18 @@ import enTranslations from '@shopify/polaris/locales/en.json';
 import type { AppProps } from 'next/app';
 import { AppProvider } from '@shopify/polaris';
 import { Layout } from '@/components/Layout';
+import "../styles/rich-text-editor.css";
```

Comment:
Does this need to be imported at this high of a level? Can it be imported within the `RichTextEditor` file itself since that's the only place it's needed?

Reply:
I believe that's what's needed, though this is my first time working on a next.js project. When I put the CSS file in the component folder I immediately get:
```
Global CSS cannot be imported from files other than your Custom <App>.
Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party
global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://nextjs.org/docs/messages/css-global
```

I don't think CSS modules can be used because there's no way to directly apply the styles to the markup outputted by the Quill component. I think this is the recommended away.
https://nextjs.org/docs/pages/building-your-application/styling/css-modules

In Remix, I just import it directly from the component. I have this in the comments of the CSS file.